src/interpreter/Interpreter.ts

```diff
@@ -50,6 +50,10 @@ export type InterpreterOptions = {
    * When interpreting a schema with discriminator set, this property will be set best by the individual interpreters to make sure the discriminator becomes an enum.
    */
   discriminator?: string;
+  /**
+   * Use this option to disable cache when interpreting schemas. This will affect merging of schemas.
```

Comment:
Do you mind clarifying a bit more what this option does? For example what cache we are talking about and when to set it to true vs false.

Reply:
Done